django/cantusdb_project/main_app/context_processors.py

```diff
@@ -2,17 +2,9 @@
 
 
 def determine_project_environment(request):
-    project_environment = os.getenv("PROJECT_ENVIRONMENT", None)
-    if not (
-        project_environment == "DEVELOPMENT"
-        or project_environment == "STAGING"
-        or project_environment == "PRODUCTION"
-    ):
-        raise ValueError(
-            "The PROJECT_ENVIRONMENT environment variable must be either "
-            "DEVELOPMENT, STAGING, or PRODUCTION. "
-            f"Its current value is {project_environment}."
-        )
+    project_environment: str = os.getenv("PROJECT_ENVIRONMENT")
```

Comment:
Don’t read the environmental variables outside of your settings file. Get the value from there, as I showed in my comment on the issue.